src/main/java/org/apache/commons/lang3/event/EventListenerSupport.java

```diff
@@ -66,9 +66,14 @@
  * </p>
  *
  * @param <L> the type of event listener that is supported by this proxy.
+ * @deprecated This class is deprecated because it does not specifically relate to event listeners. It primarily allows
+ * sequential invocation of the same method on each member of a list of objects of the same type. Modern Java features such
+ * as lambdas and executors provide more flexible and efficient ways to achieve this functionality.
+ * Consider using lambdas or executors for similar functionality.
  *
  * @since 3.0
  */
+@Deprecated
 public class EventListenerSupport<L> implements Serializable {
```

Comment:
Hi @garydgregory, could you please help review, if it is not necessary, I will close it, thanks 🙇

Reply:
@thachlp 
Please do not leave the PR description empty. Make it also contain a link to the Jira issue. 
TY!